### ISSUE TITLE
deny access on certain folders or filetypes

### ISSUE THREAD

**Issue by bax-** [h2o/h2o#756]
How would I do deny access on certain folders completely?
Using a redirect status 403 without giving a url?

How would I deny access to e.g. hidden files like .htaccess? (some applications create them by default)
Would I need ruby for that?

**Comment by utrenkner** [comment]
I have not tried it, but it should be possible using something similar to the [mruby example Setting cache-control header for certain types of files](https://h2o.examp1e.net/configure/mruby.html).

If you have only (a few) directories you want to protect you could also use the configuration shown here: https://github.com/h2o/h2o/pull/405#issuecomment-136281330

**Comment by bax-** [comment]
thanks!
Can I use the ruby handler also in global or in host context?

**Comment by utrenkner** [comment]
No, path only. See https://h2o.examp1e.net/configure/mruby_directives.html
